sdk/keyvault/azure-security-keyvault-keys/test/samples/sample2-backup-and-restore/sample2_backup_and_restore.cpp

```diff
@@ -100,8 +102,34 @@ int main()
     inFile.close();
 
     std::cout << "\t-Restore Key" << std::endl;
-    auto restoredKey = keyClient.RestoreKeyBackup(inMemoryBackup).Value;
 
+    KeyVaultKey restoredKey;
```

Comment:
Do we get a 204 ever?  According to the docs, a 204 specifies that there is no content
<html>
<body>
<!--StartFragment-->

204 No Content |   | No content, signaling that the key was permanently purged.
-- | -- | --


<!--EndFragment-->
</body>
</html>

Reply:
I seem to remember that 204 was returned for LROs when the operation had not yet completed.

Reply:
FWIW, I just checked the swagger. The 204 error is supposed to be returned when the PurgeDeletedKey API is returned.

The swagger actually specifies 204 as the only success return from a PurgeDeletedKey API.

Reply:
i don't think it matter much, purge returnes a 200 class code and the info is empty , that's why we don't check for response, if there's a 400 an exception will be thrown , so as long as we get a 200 class response it doesn't matter if it's 200 or 204 , we have no actual response returned.